eZ/Publish/Core/Persistence/Elasticsearch/Content/Search/Mapper.php

```diff
@@ -25,101 +25,19 @@
 /**
  * Mapper maps Content and Location objects to a Document object, representing a
  * document in Elasticsearch index storage.
+ *
+ * Note that custom implementations might need to be accompanied by custom mappings.
  */
-class Mapper
```

Comment:
Probably had some implementation in mind, but then ended without it :)
It is a good point, I'll change it into an interface.